src/proteus/defaults/dswx_hls.yaml

```diff
@@ -0,0 +1,70 @@
+runconfig:
+    name: dswx_hls_workflow_default
+  
+    groups:
+        pge_name_group:
+            pge_name: DSWX_HLS_PGE
+       
+        input_file_group:
+            # REQUIRED - Single HLS dataset (directory or files)
+            input_file_path:
+       
+        dynamic_ancillary_file_group:
+            # Digital elevation model
+            dem_file:
+
+            # Land cover map
+            landcover_file:
+
+            # Built up land map
+            built_up_cover_fraction_file:
+       
+        product_path_group:
+            # Directory where PGE will place results
+            product_path:
+       
+            # Directory where SAS can write temporary data
+            scratch_path:
+       
+            # Intermediate file name.  SAS writes the output product to the following file.
+            # After the SAS completes, the PGE wrapper renames the product file
+            # according to proper file naming conventions.
+            sas_output_file:
+       
+        primary_executable:
+            product_type: DSWX_HLS
+
+        hls_thresholds:
+
+            # Modified Normalized Difference Wetness Index (MNDWI) Threshold
+            wigt: 0.0124
+
+            # Automated Water Extent Shadow Threshold
+            awgt: 0.0
+
+            # Partial Surface Water Test-1 MNDWI Threshold
+            pswt_1_mndwi: -0.44
+
```

Comment:
I think exposing these parameters as well as the following few thresholds to the user does not make sense. These are very critical parameters that can completely change the algorithm performance and have been determined in results of extensive testing/calibrations. I just don't see a case where a user would want to change these values. I think we should just store them internally  as constants.

Reply:
Ok as we talked offline, we can keep this implementation as is because it could help with calibration. We can revise this decision after calibration.